Library/Homebrew/cask/url.rb

```diff
@@ -59,41 +75,29 @@ class DSL
         ).void
       }
       def initialize(
-        uri,
-        verified: nil,
-        using: nil,
-        tag: nil,
-        branch: nil,
-        revisions: nil,
-        revision: nil,
-        trust_cert: nil,
-        cookies: nil,
-        referer: nil,
-        header: nil,
-        user_agent: nil,
-        data: nil,
-        only_path: nil
+        uri, verified: nil, using: nil, tag: nil, branch: nil, revisions: nil, revision: nil, trust_cert: nil,
```

Comment:
I prefer this style when there is a `sig`, two avoid two stacks of parameter declarations, but happy to leave as-is if others disagree